src/client/javascripts/App.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    View,
+    Platform
+} from 'react-native';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        titleText: 'Yoda',
+        os: 'OS',
+      };
+
+    //For specific platform
+    if (Platform.OS === 'ios') {
+      this.state.os = 'IOS';
+    } else if (Platform.OS === 'android') {
+      this.state.os = 'Android API :' + Platform.Version;
+    }
+
+    console.log(Platform.OS);
+
+  }
+
+  render() {
+    return (
+
+            //To do about View
```

Comment:
Again, be more specific, and add a space after // and before To:

// Render the screen on View.